server/router/main.js

```diff
@@ -0,0 +1,36 @@
+module.exports = function(app) {
+  app.post('/api/client', function(req,res) {
+    // Add new client
+    const auth = req.headers['authorization'];
+    const clientToken = auth.split('=')[1];
+    if (clientToken) {
+      console.log('Store client token : ' + clientToken);
+
+      const pushManager = require('../push/push-manager.js');
```

Comment:
I'm not sure but we might have to declare to require module at top of file. (as per our guide?)
Also, I didn't like that but the |const pushManager| should be |var pushManager|. (as per our guide?)